crypto/src/main/java/org/web3j/crypto/TransactionDecoder.java

```diff
@@ -20,7 +20,7 @@ public static RawTransaction decode(String hexTransaction) {
         BigInteger value = ((RlpString) values.getValues().get(4)).asPositiveBigInteger();
         String data = ((RlpString) values.getValues().get(5)).asString();
         if (values.getValues().size() > 6) {
-            byte v = ((RlpString) values.getValues().get(6)).getBytes()[0];
+            byte[] v = ((RlpString) values.getValues().get(6)).getBytes();
```

Comment:
Can we test this too please - that we can decode a transaction that was encoded with a chainId > 256